### PR TITLE
fbe: assert that no gnome-initial-setup processes are lingering

### PR DESCRIPTION
**Pull request by wjt**
https://phabricator.endlessm.com/T24418

### CODE REVIEW

File: tests/_fbe.pm [hunk 1]
```diff
@@ -59,6 +59,11 @@ sub run {
     # desktop should be visible
     check_desktop_clean();
     save_screenshot();
+
+    # Check there are no remaining processes owned by gnome-initial-setup
+    $self->root_console(tty=>4);
```
Comment (wjt) [inline tests/_fbe.pm]: Doesn't work because `$self` is not a subclass of `installedtest`. Hm.

Comment (pwithnall) [reply tests/_fbe.pm]: I missed that in my review. Thinking…

Comment (pwithnall) [reply tests/_fbe.pm]: Maybe do something like `_enable_coredumps.pm`?

**Comment by wjt** [comment]
https://openqa.dev.endlessm.com/tests/7284#step/_post_fbe/4 it works

**Comment by pwithnall** [comment]
Ta.

**Pull request merged**